furusiya/game.py

```diff
@@ -1,3 +1,6 @@
+from model.systems.system import ComponentSystem
```

Comment:
You see, the one reason I kept everything in `Game` as `None` was that it was imported everywhere. If we import Game in ComponentSystem in the future, it'll be an inevitable cyclic import.

Reply:
I think `Game` is turning into a global-variable anti-pattern. I like how this at least suggests what the variable types are. I don't know how we can handle cyclical imports cleanly in the future.

This small change fixed quite a lot of tests that failed. In hindsight, it doesn't makes sense to create an empty `Game` instance and have `main` call `init_game` to set everything. That looks like object envy (just setting `Game` properties in `main`).